action.yml

```diff
@@ -0,0 +1,39 @@
+name: publish-cmsis-example
+description: GitHub action to publish CMSIS example to keil.arm.com
+inputs:
+  branch:
+    description: "Branch to run on"
+    required: false
+    default: ${{ github.head_ref }}
+  project-file:
+    description: "Project to import"
+    required: true
+  dry-run:
+    description: "Whether to just complete a dry-run. This can be useful to verify a project is convertable and uploadable without actually making it available on keil.arm.com"
```

Comment:
```suggestion
    description: "Whether to just complete a dry-run. This can be useful to verify a project is publishable without actually making it available on keil.arm.com"
```